### ISSUE TITLE
There is no objectdetection folder and i run the demo ,if there is no smoke or fire ,it will be wrong

### ISSUE THREAD

**Issue by senlin-ali** [RashadGarayev/FireDetection#1]

**Comment by RashadGarayev** [comment]
Please install this  https://github.com/tensorflow/models .And read installation.clone this project and add all file,folder to tensorflow/models/tree/master/research/object_detection/ folder .after run python3 object-detection-webcam.py file.

**Comment by senlin-ali** [comment]
Thanku very much!! I saw this paper before and i want to know if  this can detect fire images and fire coloured objects correctly

**Comment by RashadGarayev** [comment]
Thank you.This model trained Faster Rcnn inseption-pets ,tested-never fails. Simply using tensorflow object detection api you can assign objects with your own models instead of native ssd, mask rcnn object detection model  or other.

**Comment by tl96** [comment]
export_inference_graph.py this file not found，hope to answer,Tanks

**Comment by RashadGarayev** [comment]
> 
> 
> export_inference_graph.py this file not found，hope to answer,Tanks
I added that file to a folder

**Comment by RashadGarayev** [comment]
> 
> 
> Thanku very much!! I saw this paper before and i want to know if this can detect fire images and fire coloured objects correctly

Yes, you can use it easily. Only some light areas can be recognized as fire. You can use segmentation, which is a more ideal tool for this.
